.travis.yml

```diff
@@ -22,3 +23,10 @@ addons:
 script:
   - bazel clean && bazel build //...
   - bazel clean && bazel test //...
+
+  # Verify that all source files are correctly formatted.
+  - gofmt -d -e -l .
```

Comment:
If you are adding linting, do you wanna add `buildifier`?

https://github.com/bazelbuild/rules_docker/blob/master/.travis.yml#L23-L26

Can you also add `--test_output=error` to the `bazel test` above?